home/home_2.1/TestCases.txt

```diff
@@ -0,0 +1,44 @@
+Test Case 1:
+Step 1: User starts a program; Expected Result: Program starts
+Step 2: User should enter a name "Alice"; Expected Result: Program should display "Hello, Alice"
```

Comment:
you can specify even more details. Let's say precondition is app binary name is hello_app. Then we can write test case like:
Step 1: Execute command `./hello_app Alice`  ER: Program output is "Hello, Alice"